src/shared/locales/en/translation.json

```diff
@@ -600,6 +605,25 @@
         "nodeChangeError": "Error changing node",
         "nodeChangeErrorExplanation": "There was an error changing node. Please try again."
     },
+    "sweeps": {
+        "recoverLockedFunds": "Recover locked funds",
+        "recover": "Recover",
+        "pressButtonBelow": "Press the button below to recover your locked funds.",
+        "lockedFundsDetected": "Trinity has detected locked funds in your account.",
+        "lockedFundsRecovery": "Locked Funds Recovery",
+        "aboutLockedFunds": "About Locked Funds",
+        "lockedFundsRecoveryExplanation": "Funds have been detected on one or more addresses that have already been spent from. Spending from an address more than once is insecure.",
+        "lockedFundsToolExplanation": "This tool will attempt to salvage those funds in a way that reduces the risk involved in spending more than once. Recovery cannot be guaranteed.",
+        "lockedFundsOwnRiskExplanation": "Use this tool at your own risk.",
+        "transferYourFunds": "Transfer your funds",
+        "transferYourFundsExplanation": "Trinity will attempt to recover your funds by moving them to a new address, but it may take some time. Press continue to commence the sweep.",
+        "doNotCloseTrinity": "Do not close Trinity or turn off your computer during this process.",
+        "fundsUnlockedExplanation": "Your funds have been successfully unlocked. Promote the transactions to help them confirm.",
+        "tryAgain": "Try again",
+        "sweep": "Sweep",
+        "fromLockedAddress": "from the locked address",
+        "toSafeAddress": "to the safe address"
```

Comment:
If possible we should use [interpolation](https://www.i18next.com/translation-function/interpolation) or the [`Trans` component](https://react.i18next.com/latest/trans-component) here so that translators have better context. I can make a PR for this

Reply:
PR: #2452